Source/Core/DolphinQt/Achievements/AchievementHeaderWidget.cpp

```diff
@@ -93,6 +115,29 @@ void AchievementHeaderWidget::UpdateData()
   }
 
   AchievementManager::PointSpread point_spread = AchievementManager::GetInstance()->TallyScore();
+  const AchievementManager::BadgeStatus& game_badge =
```

Comment:
Same comment as the player badge.  This should only happen if `RA_BADGES_ENABLED` is set